de-lib/src/main/java/org/iplantc/de/client/models/webhooks/Webhook.java

```diff
@@ -0,0 +1,34 @@
+package org.iplantc.de.client.models.webhooks;
+
+import org.iplantc.de.client.models.HasId;
+
+import java.util.List;
+
+/**
+ * Created by sriram on 9/21/17.
+ *
+ * A model that describes a webhook
+ */
+public interface Webhook extends HasId {
+    //In future webhook types will come out of a service connected to a database
+    String SLACK = "Slack";
+
+    void setId(String id);
```

Comment:
It doesn't look like `setId` is being used.
Maybe it's better to remove it until we really need to change the IDs?